src/components/spaces/organisms-space.tsx

```diff
@@ -45,10 +45,18 @@ export class OrganismsSpaceComponent extends BaseComponent<IProps, IState> {
       row.setShowProteinDNA(!row.showProteinDNA);
   }
 
+  private handleSetSelectStartPercent = (percent: number) => {
+    const { organisms } = this.stores;
+    // set on both simultaneously
+    organisms.rows[0].setProteinSliderStartPercent(percent);
+    organisms.rows[1].setProteinSliderStartPercent(percent);
```

Comment:
Nice example of synchronization here! I think I see why new mice don't match position immediately - when a new mouse is added, a fresh row is created and wipes the position set here.

Reply:
This ought to be fixed now in #38